api/src/main/java/org/pmiops/workbench/cohortbuilder/CohortQueryBuilder.java

```diff
@@ -179,15 +180,23 @@ public QueryJobConfiguration buildRandomParticipantQuery(
         .build();
   }
 
-  // implemented for use with the Data Set Builder. Please remove if this does not become the
-  // preferred solution
-  // https://docs.google.com/document/d/1-wzSCHDM_LSaBRARyLFbsTGcBaKi5giRs-eDmaMBr0Y/edit#
+  // This method supports the Data Set Builder, for further details see:
+  // https://docs.google.com/document/d/1-wzSCHDM_LSaBRARyLFbsTGcBaKi5giRs-eDmaMBr0Y/edit
   public QueryJobConfiguration buildParticipantIdQuery(ParticipantCriteria participantCriteria) {
+    return buildUnionedParticipantIdQuery(ImmutableList.of(participantCriteria));
+  }
+
+  public QueryJobConfiguration buildUnionedParticipantIdQuery(
```

Comment:
nit: I think it's a little confusing that this has a different function name from the one above. The behavior is essentially the same but the different name makes it sound like this one is doing something more.

Reply:
I named it differently because I don't think it's obvious from context whether this is a union or intersection when scaled to multiple parameters. In general I also prefer not to overload if there's any doubt about semantics